conans/client/installer.py

```diff
@@ -462,7 +462,7 @@ def _handle_node_editable(self, node, profile_host, profile_build, graph_lock):
         # Get source of information
         conanfile = node.conanfile
         ref = node.ref
-        package_layout = self._cache.package_layout(ref)
+        package_layout = self._cache.get_ref_layout(self._cache.get_latest_rrev(ref))
```

Comment:
This is probably wrong, editables are not in the cache, asking for the layout of the latest revision in teh cache doesn't make sense.
In the past, ``cache.package_layout()`` could return a specific EditableLayout, but that doesn't seem the case anymore. Please check this